src/sql/src/parsers/tql_parser.rs

```diff
@@ -123,22 +125,17 @@ impl<'a> ParserContext<'a> {
         Ok(TqlParameters::new(start, end, step, lookback, query))
     }
 
-    fn find_next_delimiter_token(parser: &mut Parser) -> Token {
-        let mut n: usize = 0;
-        while !(Self::is_comma(&parser.peek_nth_token(n).token)
-            || Self::is_rparen(&parser.peek_nth_token(n).token))
-        {
-            n += 1;
+    pub fn is_delimiter_token(token: &Token, delimiter_token: &[Token]) -> bool {
```

Comment:
nit: should it be `delimiter_tokens`?

Reply:
👍 nice catch

@evenyag suggests replacing this function with`slice::contains`